delivery/templates/youtube.ftl

```diff
@@ -1,4 +1,5 @@
 <#import "/templates/system/common/crafter.ftl" as crafter />
+<script src = "http://www.youtube.com/player_api"></script>
```

Comment:
Should only be imported once despite the plugging being used multiple times on a page

Reply:
I made some enhancements for a better handling of multiple videos in the same page.